docs/3.0rc/api-ref/rest-api/index.mdx

```diff
@@ -1,18 +1,9 @@
 ---
+title: Cloud API Overview
 description: Prefect REST API for interacting with the orchestration engine and Prefect Cloud.
-tags:
-    - REST API
-    - Prefect Cloud
-    - Prefect server
-    - curl
-    - PrefectClient
-    - Requests
-    - API reference
 ---
 
-# REST API
-
-The Prefect REST API is used for communicating data from clients to a Prefect server so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
+The Prefect API is organized around REST. It is used for communicating data from clients to a Prefect server so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
```

Comment:
```suggestion
The Prefect API is organized around REST. It is used for communicating data from clients to a self-hosted Prefect server instance so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
```

Reply:
Updated for consistency elsewhere. We might want to change how we refer to this. If so, I think we should do so universally in a single PR.

Reply:
Agreed that we should change it and that it should be universal.